examples/opf/tools/sp_plotter.py

```diff
@@ -330,17 +330,16 @@ def testSPNew():
   
   
   # Setup a SP
-  sp = FDRCSpatial2.FDRCSpatial2(
```

Comment:
@chetan51 I didn't manage run these examples simply because they require previous data files that aren't in repository (`~/Desktop/ExperimentResults/Basil100x21.csv`, for example):

```
davidragazzi@davidragazzi-virtual-machine ~/Desktop/nupic $ python examples/opf/tools/sp_plotter.py
Traceback (most recent call last):
  File "examples/opf/tools/sp_plotter.py", line 572, in <module>
    testSPFile()
  File "examples/opf/tools/sp_plotter.py", line 472, in testSPFile
    file = open(inputFile, 'rb')
IOError: [Errno 2] No such file or directory: '~/Desktop/ExperimentResults/Basil100x21.csv'
```

Are you sure that this example is crucial and is updated? It seems that is quite old and a few or nobody really run this.